housekeeper/cli/get.py

```diff
@@ -105,22 +105,22 @@ def version_cmd(context, bundle_name, json, version_id, verbose):
 @click.option("-v", "--version", type=int, help="filter by version of the bundle")
 @click.option("-V", "--verbose", is_flag=True, help="print additional information")
 @click.option("-j", "--json", is_flag=True, help="Output to json format")
+@click.option("-c", "--compact", is_flag=True, help="Print a compact view")
 @click.argument("bundle", required=False)
 @click.pass_context
-def files_cmd(context, tags: List[str], version: int, verbose: bool, bundle: str, json: bool):
+def files_cmd(context, tags: List[str], version: int, verbose: bool, bundle: str, json: bool, compact: bool):
```

Comment:
Whatever we choose the option should be available for `get bundle` and `get version` as well

Reply:
Does the feature have a use-case for versions and bundles?

Reply:
`housekeeper -c ./config get version -i 2 -v`